src/app/ngrx/reducers/selectors/export.selectors.ts

```diff
@@ -75,29 +75,28 @@ export const toCsvArray = (downloads: ExportData[], dateFormat: Function): strin
     const hasTotalCol = downloads.length > 1 && downloads[1].total;
     const hasDataCols = !!downloads[0].data;
     const csvRows = [
-      hasGuidCol ? 'Title' : '""',
+      hasGuidCol ? 'Title' : '',
       ...(hasGuidCol ? ['GUID'] : []),
       ...(hasPubDateCol ? ['Release Date'] : []),
       ...(hasTotalCol ? ['Total'] : []),
       ...(hasDataCols ? downloads[0].data.map(col => dateFormat(new Date(col[0]))) : [])
     ];
     return [csvRows].concat(downloads.map(d => [
-      `"${d.label}"`,
+      d.label,
       ...(hasGuidCol ? [d.guid] : []),
       ...(hasPubDateCol ? [d.publishedAt && dateUtil.ISODate(d.publishedAt) || ''] : []),
-      ...(hasTotalCol ? [d.total] : []),
+      ...(hasTotalCol ? [d.total.toString()] : []),
       ...(hasDataCols ? d.data.map(col => col[1].toString()) : [])
     ]));
   }
 };
 
```

Comment:
changes here are to only quote values where needed for CSV exports. Google Sheets and CSV share the toCSVArray, the join is just for CSV downloads.